src/components/Advertisements/skeleton/AdvertisementSkeleton.tsx

```
@@ -20,23 +22,33 @@ import styles from 'style/app-fixed.module.css';
+
   return [...Array(6)].map((_, index) => (
-    <div
-      key={index}
-      className={styles.itemCard}
-      data-testid={`skeleton-${index + 1}`}
+    <ErrorBoundaryWrapper
+      fallbackErrorMessage={tErrors('defaultErrorMessage')}
+      fallbackTitle={tErrors('title')}
+      resetButtonAriaLabel={tErrors('resetButtonAriaLabel')}
+      resetButtonText={tErrors('resetButton')}
     >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)